src/main/java/nl/tudelft/ti2206/group9/audio/SoundEffectPlayer.java

```diff
@@ -45,17 +42,11 @@ public SoundEffectPlayer(final String soundPath) {
     @Override
     protected final void initializeAudio(final String source) {
         try {
-            audioClip = new AudioClip(new File(source).toURI().toURL()
-                    .toString());
-        } catch (MalformedURLException mue) {
```

Comment:
No, the `MalformedURLException` is handled in the class `Resource`. In the new code, `Resource.getURIString(source)` is called, which handles any malformed URLs.
The `MediaException` can still be thrown by the AudioClip, and is thus still being caught here in `SoundEffectPlayer`.

<review